.jenkinsci/debug-build.groovy

```diff
@@ -66,7 +66,7 @@ def doDebugBuild(coverageEnabled=false) {
     + " -v /tmp/${GIT_COMMIT}-${BUILD_NUMBER}:/tmp/${GIT_COMMIT}") {
 
     def scmVars = checkout scm
-    def cmakeOptions = ""
+    def cmakeOptions = "-DCMAKE_CXX_COMPILER=${WORKSPACE}/.jenkinsci/helpers/build.sh"
```

Comment:
are you sure we do not need `DCMAKE_C_COMPILER` also ?

Reply:
how it will work if `fuzzingEnabled` options is set to true?
because it adds
`cmakeOptions += " -DCMAKE_C_COMPILER=clang-6.0 -DCMAKE_CXX_COMPILER=clang++-6.0 -DFUZZING=ON "`

Reply:
I think yes, because compiler tracks only "g++" operations